geode-common/src/main/java/org/apache/geode/internal/inet/LocalHostUtil.java

```diff
@@ -242,6 +242,10 @@ public static InetAddress getAnyLocalAddress() {
     return new InetSocketAddress(0).getAddress();
   }
 
+  public static boolean isWildcardAddress(String address) {
+    return (address != null && (address.equals("0.0.0.0") || address.equals("::")));
```

Comment:
How about simplifying this as follows?:
return "0.0.0.0".equals(address) || "::".equals(address);

Reply:
Good catch! I have just added a commit fixing this. Thanks!